src/components/GridCard.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import Link from "gatsby-link";
+import withStyles from "elevate-ui/withStyles";
+
+const GridCard = ({ classes, text, color }) => {
+  return (
+    <div className={classes.root}>
+      <div className={classes.inner}>
+        <div className={classes.title}>{text}</div>
```

Comment:
Looks like `title` is never used, can we combine classes.inner and classes.title into a single div/className?